times.go

```diff
@@ -90,10 +90,14 @@ func CustomRelTime(a, b time.Time, albl, blbl string, magnitudes []RelTimeMagnit
 		diff = a.Sub(b)
 	}
 
-	n := sort.Search(len(magnitudes), func(i int) bool {
+	lenMag := len(magnitudes)
```

Comment:
Is `lenMag` neccessary? It doesn't seem to be, and I think the code would be more readable if you just used `len(magnitudes)` everywhere. I think it'd be just:

```Go
if n >= len(magnitudes) {
	n = len(magnitudes) - 1
}
```